examples/app.component.ts

```diff
@@ -0,0 +1,62 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Atom, atom, Derivable, derivation } from '@politie/sherlock';
+import { List, Map } from 'immutable';
+import { bohemia } from './text/bohemia';
+import { redHeadedLeague } from './text/red-headed-league';
+import { UtilsService } from './utils.service';
+
+@Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppComponent {
+
+    constructor(private utils: UtilsService) { }
+
+    private readonly books = { bohemia, redHeadedLeague };
+
+    /**
+     * Atoms are the basic building blocks of this reactive application. They are mutable references to immutable values.
+     * Atoms represent the ground truth from which the total application state is derived.
+     */
+    private readonly currentBook$: Atom<List<string>> = atom(this.books.bohemia);
+    readonly currentFontSize$: Atom<number> = atom(this.utils.getAvailableSizes()[0]);
+    readonly currentPageNumber$: Atom<number> = atom(0);
+
+    /**
+     * Derivations are calculated derived state based on Atoms or other Derivations. They can be created by calling #derive
+     * on an Atom or other derivation, or can be made with the `derivation()` function. The latter automatically registers
```

Comment:
Both automatically register the `Derivables` they are dependent on,.. Not only `derivation()`.
The only difference is that `.derive()` unpacks the `Derivable` it is called upon for you..